TaskList/app/views/layouts/_task_printout.html.erb

```diff
@@ -0,0 +1,42 @@
+<form method="post"
+  <% if @method == :delete %>
+    action="/delete_confirmation"
+  <% end %>
+>
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <td>Task</td>
+        <td>Description</td>
+        <td>Completed</td>
+      </tr>
+    </thead>
+    <tbody>
+      <% @tasks.each do |task| %>
+      <% completed = !task.completed_date.nil? %>
+      <tr>
+        <td <% if completed %>
+          class="strikethru"
```

Comment:
I wonder if you could add the class to the `tr` instead of each `td` and if that would give you the same or a similar result